src/wallet/crypter.h

```diff
@@ -185,6 +187,34 @@ class CCryptoKeyStore : public CBasicKeyStore
             mi++;
         }
     }
+    virtual bool AddCryptedSpendingKey(const libzcash::PaymentAddress &address, const std::vector<unsigned char> &vchCryptedSecret);
+    bool AddSpendingKeyPaymentAddress(const libzcash::SpendingKey &key, const libzcash::PaymentAddress &address);
+    bool HaveSpendingKey(const libzcash::PaymentAddress &address) const
+    {
```

Comment:
FWIW, I think consistency of putting all the implementations together is more important than consistency with upstream.